picard/ui/aboutdialog.py

```diff
@@ -87,8 +89,7 @@ def load(self):
 <a href="https://www.flaticon.com/authors/smashicons">Smashicons</a>
 from <a href="https://www.flaticon.com">www.flaticon.com</a>""")
 
-        text = _("""<p align="center"><span style="font-size:15px;font-weight:bold;">MusicBrainz Picard</span><br/>
-Version %(version)s</p>
+        text = _("""<p align="center">Version %(version)s</p>
```

Comment:
@phw I think we should explode this string in smaller bits, since everytime a little change happens it needs to be re-translated.
See https://github.com/phw/picard/pull/4

Reply:
Yes, this is a big improvement, thanks a lot.